src/main/java/net/minestom/server/event/EventNode.java

```diff
@@ -332,6 +333,13 @@ default void removeChildren(@NotNull String name) {
     @Contract(value = "_ -> this")
     @NotNull EventNode<T> removeListener(@NotNull EventListener<? extends T> listener);
 
+    /**
+     * Get all listeners registered to this node.
+     * @return A collection of listeners.
+     */
+    @Contract(value = "-> this")
+    @NotNull Collection<EventListener<T>> getListeners();
```

Comment:
The `Contract` is wrong. Also `listeners()` would be a better name

Reply:
I removed the contract and change the method's name.